halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/v2/KubectlServiceProvider.java

```diff
@@ -39,14 +44,31 @@
 @Component
 public class KubectlServiceProvider extends SpinnakerServiceProvider<AccountDeploymentDetails<KubernetesAccount>> {
   @Autowired
+  @Qualifier("kubernetesV2ClouddriverService")
```

Comment:
What are these for?

Reply:
This is because I extended KubernetesV2ClouddriverService for three other services so spring cannot choose what to autowire here without the qualifier.